servicecontrol/troubleshooting.md

```diff
@@ -131,20 +131,19 @@ or
 System.InvalidOperationException: Cannot modify indexes while indexing is in progress (already waited full minute). Try again later
```

Comment:
It seems weird that this is a full log line whereas the first error is just a fragment.